src/components/ExportLicenseAsCSVModal/ExportLicenseAsCSVModal.js

```diff
@@ -0,0 +1,175 @@
+            />
+          ))
+        }
+        </Layout>
+      </>
+    );
+  }
+
+  renderExportLicenseAsCSVModal = () => {
+    const { licenseInformation, terms, termNames } = this.state;
+    const footer = (
+      <ModalFooter>
+        <Button
+          buttonStyle="primary"
+          disabled={Object.values({ ...licenseInformation, ...terms, ...termNames }).includes(true) !== true}
+          id="export-licenses-modal-save-button"
+          onClick={() => {
+            const payload = {
+              ids: this.props.selectedLicenses,
+              include: { ...pickBy(licenseInformation), ...{ 'customProperties': pickBy(termNames) } },
+              terms: pickBy(terms)
+            };
+
+            this.props.onCompareLicenseTerms(payload)
+              .finally(() => this.props.onClose());
+          }}
+        >
+          <FormattedMessage id="stripes-components.saveAndClose" />
+        </Button>
+        <Button
+          buttonStyle="default"
+          id="export-licenses-modal-cancel-button"
+          onClick={this.props.onClose}
+        >
+          <FormattedMessage id="stripes-components.cancel" />
+        </Button>
+      </ModalFooter>
+    );
+
+    return (
+      <Modal
+        dismissible
+        footer={footer}
+        id="export-licenses-modal"
+        label={<FormattedMessage id="ui-licenses.exportLicenses" />}
+        onClose={this.props.onClose}
+        open
+        size="medium"
+      >
+        <Layout className="padding-bottom-gutter">
+          <FormattedMessage id="ui-licenses.exportLicensesModal.message" />
+        </Layout>
+        <Layout className="padding-bottom-gutter">
+          <Checkbox
+            checked={Object.values({ ...licenseInformation, ...terms, ...termNames }).includes(false) !== true}
+            label={<strong><FormattedMessage id="ui-licenses.exportLicensesModal.selectAll" /></strong>}
+            onChange={this.toggleSelectAll}
+            value="selectAll"
+          />
+        </Layout>
+        <div className={css.separator} />
+        {this.renderCheckboxesList('licenseInformation')}
+        {this.renderCheckboxesList('terms')}
+        <Layout className="padding-start-gutter">
+          {this.renderCheckboxesList('termNames')}
+        </Layout>
+      </Modal>
+    );
+  }
+
+  toggleSelectAll = (e) => {
+    const { checked } = e.target;
+
+    this.setState(() => ({
+      licenseInformation: mapValues(this.licenseInformation, () => checked),
+      terms: mapValues(this.terms, () => checked),
+      termNames: mapValues(this.termNames, () => checked)
+    }));
+  }
+
+  toggleSelectSection = (e, section) => {
+    const { checked } = e.target;
+
+    this.setState(() => ({
+      [section]: mapValues(this[section], () => checked),
+    }));
+  }
+
+  updateSelection = (e, section) => {
+    const { checked, name } = e.target;
+    this.setState(prevState => ({
+      [section]: { ...prevState[section], [name]: checked }
+    }));
+  };
+
```

Comment:
Keep the `render***` methods as the last ones in the class. I guess this lint rule got toggled off or didn't work or something 🤷‍♂